solana/programs/example-native-token-transfers/src/instructions/release_inbound.rs

```diff
@@ -40,6 +41,13 @@ pub struct ReleaseInbound<'info> {
     pub mint: InterfaceAccount<'info, token_interface::Mint>,
 
     pub token_program: Interface<'info, token_interface::TokenInterface>,
+
+    /// CHECK: the token program checks if this indeed the right authority for the mint
+    #[account(
+        mut,
+        address = config.custody
+    )]
+    pub custody: InterfaceAccount<'info, token_interface::TokenAccount>,
```

Comment:
Given this change and seeing how custody is now used both in locking and in mint/burn mode, the [comment in initialize](https://github.com/wormhole-foundation/example-native-token-transfers/blob/bf49cabe9f539fbf284925ab81d1e28722b75050/solana/programs/example-native-token-transfers/src/instructions/initialize.rs#L71-L72) about it not being used in the latter is now outdated.

Reply:
good catch, updated the comment